### PR TITLE
Blog post on .NET WebAssembly ecosystem status

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (brooksmtownsend) [inline content/english/blog/dotnet_are_we_webassembly_yet.md:42]: I'd suggest adding language annotations like `terminal` or `shell` here so that the backticked sections are distinguished

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (brooksmtownsend) [inline content/english/blog/dotnet_are_we_webassembly_yet.md]: This is a good opportunity for an `aside` like in the other posts

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (brooksmtownsend) [inline content/english/blog/dotnet_are_we_webassembly_yet.md]: After `There is also absolutely no sign of work beginning in .NET on support for the component model`, this is a great time to link back to the ubiquity blog post and tie this post with a nice pretty bow of "We know the component model isn't there yet, and things are very runtime specific. This is the way it works today, and the team maintaining wasmCloud encourages this standards work so that WebAssembly developers don't have difficulty with different language toolchains"

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (brooksmtownsend) [inline content/english/blog/dotnet_are_we_webassembly_yet.md]: Will need to change this for the current date but that can wait until an approve or two

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
Would you mind cropping the picture so that there isn't the purple space at the bottom, or is that the _official_ .net logo for Wasm?

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (autodidaddict) [reply content/english/blog/dotnet_are_we_webassembly_yet.md:42]: when I add those, the rendering is completely illegible - dark red on black with 3 blanks per line.

**Review by brooksmtownsend — APPROVED** [review]
This is fantastic 🚀

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (brooksmtownsend) [reply content/english/blog/dotnet_are_we_webassembly_yet.md:42]: Gross. Disregard then!

File: content/english/blog/dotnet_are_we_webassembly_yet.md
Comment (ricochet) [inline content/english/blog/dotnet_are_we_webassembly_yet.md]: Nit: With the gc proposal, significant chunks of the gc code in the wasm module is eliminated. Modules that build in the interpreter are going to be the long-pole in the tent.

Aside to your aside: I bet languages like tinygo+gc proposal will be smaller in size (likely not memory) than a C++ app that heavily uses smart pointers.

**Review by ricochet — APPROVED** [review]
This was a really fun read

**Pull request merged**
